### PR TITLE
[SMALL] PR for version bump for PyPi Release

### PR DESCRIPTION
**Pull request by chaithyagr**
This PR just has a minor code for version bump to 0.2.0.

Also, I have cleaned up AUTHOR_EMAIL which was set to XXX, as the emails are present in the AUTHOR field.

Edit: Just saw that gpuNUFFT has merged my codes, so I am updating the readme too.

### CODE REVIEW

**Comment by chaithyagr** [comment]
I just noticed my PR is merged on gpuNUFFT. 
I have updated README too accordingly.
Updating description

**Comment by philouc** [comment]
congrats @chaithyagr for your PR merged on gpuNUFFT

**Comment by chaithyagr** [comment]
Thank you @philouc !

File: README.rst [hunk 1]
```diff
@@ -38,7 +38,7 @@ Special Installations
 
 For faster NUFFT operation, pysap-mri uses gpuNUFFT, to run the NUFFT on GPU. To install gpuNUFFT, please use:
 
-``pip install git+https://github.com/chaithyagr/gpuNUFFT@python_support``
+``pip install git+https://github.com/andyschwarzl/gpuNUFFT``
```
Comment (zaccharieramzi) [inline README.rst:41]: yay !

**Comment by chaithyagr** [comment]
Looks like we need the author email after all for documentation builds.
`OSError: Missing mandatory fields {'AUTHOR_EMAIL'} in '/home/travis/build/chaithyagr/pysap-mri/install/mri/info.py'.`
I will split the current author field and place the email here..

**Pull request merged**